backend/sumo-championship/src/main/java/com/sumoc/sumochampionship/db/season/Category.java

```diff
@@ -26,17 +26,20 @@ public class Category {
     private Long id;
 
     private String name;
+
+    // Added maxAge - TODO: Discuss
```

Comment:
We definitely need `maxAge`. I'd rather discuss `minAge`.

Reply:
For now I will leave it. If front-end decide to not implement minAge as option of the Category, I will delete it